src/components/Pages/Recipes.js

```diff
@@ -56,12 +56,15 @@ function Recipes() {
             });
             const expirationTime = parseISO(recipes[i].expiration_time);
             const expirationTimeFormatted = format(expirationTime, "PPp");
-            console.log(isAfter(expirationTime, new Date()));
-            const expirationTimeAgo = isAfter(expirationTime, new Date())
-              ? formatDistance(new Date(), expirationTime, { addSuffix: true })
-              : formatDistance(expirationTime, new Date(), {
-                  addSuffix: true,
-                });
+            console.log(
+              isAfter(expirationTime, new Date()),
+              recipes[i].task_name
+            );
+            const expirationTimeAgo = formatDistance(
+              expirationTime,
+              new Date(),
```

Comment:
https://date-fns.org/v2.0.0-alpha.26/docs/formatDistance#syntax

for ref - second parameter is the BASE DATE!